workbench/accounts/features.py

```diff
@@ -41,6 +41,7 @@ class FEATURES(str, Enum):
     PLANNING = "PLANNING"
     PRIDE = "PRIDE"
     PROJECTED_GROSS_MARGIN = "PROJECTED_GROSS_MARGIN"
+    WEBHOOKS = "WEBHOOKS"
```

Comment:
What's the reason for this? I cannot find any further use of this feature flag.

Reply:
I thought these are needed to display the model admin